client/lib/plans/plans-list.js

```diff
@@ -374,19 +374,37 @@ const getPlanBusinessDetails = () => ( {
 const getPlanJetpackSecurityDailyDetails = () => ( {
 	group: constants.GROUP_JETPACK,
 	type: constants.TYPE_SECURITY_DAILY,
-	getTitle: () => translate( 'Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+	getTitle: () =>
+		getJetpackCROActiveVersion() === 'v2'
+			? translate( 'Jetpack Security {{em}}Daily{{/em}}', { components: { em: <em /> } } )
+			: translate( 'Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+	getButtonLabel: () =>
+		getJetpackCROActiveVersion() === 'v2'
+			? translate( 'Get Security {{em}}Daily{{/em}}', { components: { em: <em /> } } )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Security {{em}}Daily{{/em}}' )` *ES Score: 9*